runelite-client/src/main/java/net/runelite/client/util/LinkBrowser.java

```diff
@@ -56,30 +56,67 @@ public static boolean browse(final String url)
 			return false;
 		}
 
+		if (OSType.getOSType() == OSType.Linux && attemptCommandBrowse("xdg-open " + url))
+		{
+				log.debug("Opened browser through xdg-open to {}", url);
+				return true;
+		}
+
+		if (attemptDesktopBrowse(url))
+		{
+			log.debug("Opened browser through Desktop#browse to {}", url);
+			return true;
+		}
+
+		showMessageBox("Unable to open link. Press 'OK' and link will be copied to your clipboard.", url);
+		return false;
+	}
+
+	private static boolean attemptCommandBrowse(String command)
```

Comment:
you should store if xdg-open exists in static variable or something instead of trying it all the time on linux